packages/flutter/lib/src/painting/image_provider.dart

```diff
@@ -486,7 +486,8 @@ class NetworkImage extends ImageProvider<NetworkImage> {
       request.headers.add(name, value);
     });
     final HttpClientResponse response = await request.close();
-    if (response.statusCode != HttpStatus.ok)
+    // ignore: deprecated_member_use
+    if (response.statusCode != HttpStatus.OK)
```

Comment:
nit: prefer putting the ignores on the actual line that has the problem

Reply:
Done.